src/workerd/jsg/setup.h

```diff
@@ -617,10 +617,10 @@ class Isolate: public IsolateBase {
   };
 
   // The func must be a callback with the signature: void(jsg::Lock&)
```

Comment:
Don't forget to update comment now that this can return values.
Also should probably add a comment about being careful from returning v8 locals trying to escape the scope

Reply:
Is it even used in this PR?

Reply:
yes, I use it to return bytes.